### PR TITLE
[Improvement] Update doc to configure MapReduce client

### PR DESCRIPTION
**Pull request by frankliee**
### What changes were proposed in this pull request?
 Update doc to configure MapReduce client.

### Why are the changes needed?
The previous version only has Spark configuration.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
No, just doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,24 @@ After apply the patch and rebuild spark, add following configuration in spark co
   spark.dynamicAllocation.enabled true
   ```
 
+## Deploy MapReduce Client
+
+1. Add client jar to Hadoop classpath, e.g., HADOOP_HOME/share/hadoop/mapreduce/
+
+The jar for MapReduce is located in <RSS_HOME>/jars/client/mr/rss-client-mr-XXXXX-shaded.jar
```
Comment (jerqi) [inline README.md:144]: We need update jar to HDFS

File: README.md [hunk 2]
```diff
@@ -137,6 +137,24 @@ After apply the patch and rebuild spark, add following configuration in spark co
...
+
+2. Update MapReduce conf to enable Firestorm, the following demo is for local storage only, eg,
+
+   ```
+   -Dmapreduce.rss.coordinator.quorum=<coordinatorIp1>:19999,<coordinatorIp2>:19999
```
Comment (jerqi) [inline README.md:149]: we use this configuration to download jar, mapreduce.job.cache.files

Comment (jerqi) [reply README.md:149]: We use this configuration `mapreduce.application.classpath` to add ClassPath

Comment (frankliee) [reply README.md:149]: Ok

File: README.md [hunk 3]
```diff
@@ -186,6 +204,15 @@ The important configuration is listed as following.
 |spark.rss.client.send.threadPool.size|10|The thread size for send shuffle data to shuffle server|
 
 
+### MapReduce Client
+
+|Property Name|Default|Description|
+|---|---|---|
+|mapreduce.rss.coordinator.quorum|-|Coordinator quorum|
+|mapreduce.rss.storage.type|-|Supports MEMORY_LOCAL, MEMORY_HDFS, LOCALFILE, HDFS, LOCALFILE_HDFS|
```
Comment (jerqi) [inline README.md:212]: Not recommend HDFS

Comment (frankliee) [reply README.md:212]: ok

File: README.md [hunk 4]
```diff
@@ -139,13 +139,14 @@ After apply the patch and rebuild spark, add following configuration in spark co
...
 
 The jar for MapReduce is located in <RSS_HOME>/jars/client/mr/rss-client-mr-XXXXX-shaded.jar
 
 2. Update MapReduce conf to enable Firestorm, the following demo is for local storage only, eg,
 
    ```
+   -Dmapreduce.job.cache.files=hdfs://your/lib/path/rss-client-mr-XXXXX-shaded.jar
    -Dmapreduce.rss.coordinator.quorum=<coordinatorIp1>:19999,<coordinatorIp2>:19999
    -Dmapreduce.rss.storage.type=MEMORY_LOCALFILE 
    -Dyarn.app.mapreduce.am.command-opts=org.apache.hadoop.mapreduce.v2.app.RssMRAppMaster
```
Comment (jerqi) [inline README.md:152]: We use this configuration mapreduce.application.classpath to add ClassPath

File: README.md [hunk 5]
```diff
@@ -137,6 +137,25 @@ After apply the patch and rebuild spark, add following configuration in spark co
...
+
+   ```
+   -Dmapreduce.job.cache.files=hdfs://your/lib/path/rss-client-mr-XXXXX-shaded.jar
+   -Dmapreduce.rss.coordinator.quorum=<coordinatorIp1>:19999,<coordinatorIp2>:19999
+   -Dmapreduce.rss.storage.type=MEMORY_LOCALFILE 
+   -Dyarn.app.mapreduce.am.command-opts=org.apache.hadoop.mapreduce.v2.app.RssMRAppMaster
+   -Dmapreduce.job.map.output.collector.class=org.apache.hadoop.mapred.RssMapOutputCollector
+   -Dmapreduce.job.reduce.shuffle.consumer.plugin.class=org.apache.hadoop.mapreduce.task.reduce.RssShuffle
+   ```
+Note that the RssMRAppMaster will automatically disable slow start (i.e., `mapreduce.job.reduce.slowstart.completedmaps=1`)
```
Comment (jerqi) [inline README.md:156]: Support Hadoop version

**Review by jerqi — APPROVED** [review]

**Pull request merged**